core/src/main/scala/scalaz/Isomorphism.scala

```diff
@@ -235,7 +235,7 @@ trait IsomorphismOrder[F, G] extends Order[F] {
   override def order(x: F, y: F): Ordering = G.order(iso.to(x), iso.to(y))
 }
 
-trait IsomorphismEnum[F, G] extends Enum[F] {
+trait IsomorphismEnum[F, G] extends Enum[F] with IsomorphismOrder[F, G] {
```

Comment:
this probably breaks bincompat

Reply:
I've added because the rules seems to be if  typeclass B is a subtype a typeclass A, then typeclass IsomorphismB would mix in IsomorphismA, for instance `IsomorphismMonoid` mixes in `IsomorphismSemigroup`. I suspected it that it could break the bincompat, I'll remove it and I'll get `IsomorphismEnum` to overrides the `order` function as well.